samples/canopen/canopen_rpdo/RPDOCanNode.hpp

```diff
@@ -0,0 +1,226 @@
+     */
+    CO_OBJ_T* getObjectDictionary();
+
+    /**
+     * Get the number of elements in the object dictionary.
+     *
+     * @return The number of elements in the object dictionary
+     */
+    uint8_t getNumElements();
+
+    /**
+     * The node ID used to identify the device on the CAN network.
+     */
+    static constexpr uint8_t NODE_ID = 2;
+
+    static constexpr uint8_t TPDO_NODE_ID = 1;
+
+private:
+    /**
+     * This sample data will be exposed over CAN through the object
+     * dictionary. The address of the variable will be included in the
+     * object dictionary and can be updated via SDO via a CANopen client.
+     * This device will then broadcast the value via a triggered PDO.
+     */
+    uint8_t sampleDataA;
+    uint16_t sampleDataB;
+
+    /**
+     * Have to know the size of the object dictionary for initialization
+     * process.
+     */
+    static constexpr uint8_t OBJECT_DICTIONARY_SIZE = 22;
+
+    /**
+     * The object dictionary itself. Will be populated by this object during
+     * construction.
+     *
+     * The plus one is for the special "end of dictionary" marker.
+     */
+    CO_OBJ_T objectDictionary[OBJECT_DICTIONARY_SIZE + 1] = {
+        /**
+         * Mandatory Identification Keys
+         */
+        {// Device Type
+         .Key = CO_KEY(0x1000, 0x00, CO_OBJ_____R_),
+         .Type = CO_TUNSIGNED32,
+         .Data = (CO_DATA) 0x00},
+        {// Error Register
+         .Key = CO_KEY(0x1001, 0x00, CO_OBJ_____R_),
+         .Type = CO_TUNSIGNED8,
+         .Data = (CO_DATA) 0x00},
+        {// Sync ID, defaults to 0x80
+         .Key = CO_KEY(0x1005, 0x00, CO_OBJ_DN__R_),
+         .Type = CO_TUNSIGNED32,
+         .Data = (CO_DATA) 0x80},
+        {// COB-ID EMCY
+         .Key = CO_KEY(0x1014, 0x00, CO_OBJ__N__R_),
+         .Type = CO_TEMCY_ID,
+         .Data = (CO_DATA) 0x80},
+        {// Heartbeat Producer
+         .Key = CO_KEY(0x1017, 0x00, CO_OBJ_D___R_),
+         .Type = CO_THB_PROD,
+         .Data = (CO_DATA) 2000},
+
+        /*
+         * Identity Object
+         * 0. Object Start Key
+         * 1. Vendor ID
+         * 2. Product Code
+         * 3. Revision Number
+         * 4. Serial Number
+         */
+        {// Identity Object
+         .Key = CO_KEY(0x1018, 0x00, CO_OBJ_D___R_),
+         .Type = CO_TUNSIGNED8,
+         .Data = (CO_DATA) 0x04},
+        {// Vendor ID
+         .Key = CO_KEY(0x1018, 0x01, CO_OBJ_____R_),
+         .Type = CO_TUNSIGNED8,
+         .Data = (CO_DATA) 0x00},
+        {// Product Code
+         .Key = CO_KEY(0x1018, 0x02, CO_OBJ_____R_),
+         .Type = CO_TUNSIGNED8,
+         .Data = (CO_DATA) 0x00},
+        {// Revision Number
+         .Key = CO_KEY(0x1018, 0x03, CO_OBJ_____R_),
+         .Type = CO_TUNSIGNED8,
+         .Data = (CO_DATA) 0x00},
+        {// Serial number
+         .Key = CO_KEY(0x1018, 0x04, CO_OBJ_____R_),
+         .Type = CO_TUNSIGNED8,
+         .Data = (CO_DATA) 0x00},
```

Comment:
```suggestion
        {// Serial number
         .Key = CO_KEY(0x1018, 0x04, CO_OBJ_____R_),
         .Type = CO_TUNSIGNED8,
         .Data = (CO_DATA) 0x00,
        },
```
Include a comma at the end of .Data so the autoformat wraps the curly bracket onto the next line. These example dictionaries should have one consistent format.